app/components/Time.js

```diff
@@ -28,44 +29,53 @@ type Props = {
   shorten?: boolean,
 };
 
-class Time extends React.Component<Props> {
-  removeEachMinuteCallback: () => void;
+function Time({ addSuffix, children, dateTime, shorten, tooltipDelay }: Props) {
+  const { auth } = useStores();
+  const [_, setMinutesMounted] = React.useState(0); // eslint-disable-line no-unused-vars
+  const callback = React.useRef();
 
-  componentDidMount() {
-    this.removeEachMinuteCallback = eachMinute(() => {
-      this.forceUpdate();
+  React.useEffect(() => {
+    callback.current = eachMinute(() => {
+      setMinutesMounted((state) => ++state);
     });
-  }
 
-  componentWillUnmount() {
-    this.removeEachMinuteCallback();
-  }
+    return () => {
+      if (callback.current) {
+        callback.current();
+      }
+    };
+  }, []);
 
-  render() {
-    const { shorten, addSuffix } = this.props;
-    let content = distanceInWordsToNow(this.props.dateTime, {
-      addSuffix,
-    });
+  let content = distanceInWordsToNow(dateTime, {
+    addSuffix,
+    locale: getLocaleForUser(auth.user),
+  });
 
-    if (shorten) {
-      content = content
-        .replace("about", "")
-        .replace("less than a minute ago", "just now")
-        .replace("minute", "min");
-    }
+  if (shorten) {
+    content = content
+      .replace("about", "")
+      .replace("less than a minute ago", "just now")
+      .replace("minute", "min");
+  }
 
-    return (
-      <Tooltip
-        tooltip={format(this.props.dateTime, "MMMM Do, YYYY h:mm a")}
-        delay={this.props.tooltipDelay}
-        placement="bottom"
-      >
-        <time dateTime={this.props.dateTime}>
-          {this.props.children || content}
-        </time>
-      </Tooltip>
-    );
+  return (
+    <Tooltip
+      tooltip={format(dateTime, "MMMM Do, YYYY h:mm a")}
+      delay={tooltipDelay}
+      placement="bottom"
+    >
+      <time dateTime={dateTime}>{children || content}</time>
+    </Tooltip>
+  );
+}
+
+function getLocaleForUser(user) {
+  if (!user) {
+    return undefined;
   }
+
+  const userLanguage = user.language.split("_")[0];
```

Comment:
This made me realise that the flowtype for `user.language` is actually wrong, it can be `null` on older accounts.

I think we can `en_US` as a default in the presenter here to fix that:

https://github.com/outline/outline/blob/develop/server/presenters/user.js#L27

Reply:
Should we use `process.env.DEFAULT_LANGUAGE` just in case?

Reply:
We can use it in the sever side, yea that's a good idea.